packages/integrations/markdoc/README.md

```diff
@@ -90,6 +90,8 @@ const { Content } = await entry.render();
 <!--Render Markdoc contents with the Content component-->
 <Content />
 ```
+> **Note**
+> The `<Content />` component renders your Markdoc content within `<article></article>`. This may affect CSS and styling of your content.
```

Comment:
Can we offer a code snippet on how to fix this? Something like:

```ts
// markdoc.config.mjs
import { defineMarkdocConfig } from '@astrojs/markdoc/config';

export default defineMarkdocConfig({
  nodes: {
    document: {
      render: null, // don't render a wrapper element
      // Defaults from `Markdoc.nodes.document`
			children: [
				'heading',
				'paragraph',
				'image',
				'table',
				'tag',
				'fence',
				'blockquote',
				'comment',
				'list',
				'hr',
			],
    }
  }
});
```

Reply:
Ooh, I didn't realize this could be circumvented. OK, now this is getting to be a thing, I'll figure out how to incorporate this.